src/Sentry/Extensibility/HubAdapter.cs

```diff
@@ -63,13 +63,6 @@ public IDisposable PushScope()
         public IDisposable PushScope<TState>(TState state)
             => SentrySdk.PushScope(state);
 
-        /// <summary>
-        /// Forwards the call to <see cref="SentrySdk"/>.
-        /// </summary>
-        [DebuggerStepThrough]
```

Comment:
we'll need to go through the deprecation thing, add some [Obsolete] attribute, telling users how to migrate
then delete it in the future on a major version release